build_with_ebpf/bad_cargo/hijack-inputs.Dockerfile

```diff
@@ -1,5 +1,5 @@
-from rust:latest
+FROM rust:latest
```

Comment:
Note: This is required because the released version of Kaniko image doesn't detect lowercase FROM. I fixed this upstream but it hasn't been released